pkg/kubelet/rkt/rkt.go

```diff
@@ -51,8 +51,8 @@ import (
 )
 
 const (
-	acVersion             = "0.6.1"
-	rktMinimumVersion     = "0.8.1"
+	acVersion             = "0.7.1"
+	supportedRktVersion   = "0.9.0"
```

Comment:
ok, added minimum version and recommended version. Fail if the version < minimum version. Print warnings if the version != recommended version.